projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,191 @@
+                                style="width: 70%"
+                                aria-valuenow="70"
+                                aria-valuemin="0"
+                                aria-valuemax="100"
+                        >
+                            70%
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <h1>ASP.Net Core</h1>
+                    <div class="progress">
+                        <div
+                                class="progress-bar"
+                                role="progressbar"
+                                style="width: 55%"
+                                aria-valuenow="25"
+                                aria-valuemin="0"
+                                aria-valuemax="100"
+                        >
+                            55%
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="container text-center mt-5">
+            <div class="row">
+                <div class="col">
+                    <h1>Git/Github</h1>
+                    <div class="progress">
+                        <div
+                                class="progress-bar"
+                                role="progressbar"
+                                style="width: 80%"
+                                aria-valuenow="70"
+                                aria-valuemin="0"
+                                aria-valuemax="100"
+                        >
+                            80%
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <h1>Docker</h1>
+                    <div class="progress">
+                        <div
+                                class="progress-bar"
+                                role="progressbar"
+                                style="width: 30%"
+                                aria-valuenow="25"
+                                aria-valuemin="0"
+                                aria-valuemax="100"
+                        >
+                            30%
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row mt-5 mb-5">
+                <h1>Network & Server</h1>
+                <div class="progress">
+                    <div
+                            class="progress-bar"
+                            role="progressbar"
+                            style="width: 20%"
+                            aria-valuenow="25"
+                            aria-valuemin="0"
+                            aria-valuemax="100"
+                    >
+                        20%
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<script src="../../../Content/Scripts/bootstrap.min.js"></script>
```

Comment:
you can also remove this local js reference, you have the external Js file in the next line.